src/realm/array_mixed.hpp

```diff
@@ -135,6 +144,7 @@ class ArrayMixed : public ArrayPayload, private Array {
     mutable ArrayString m_strings;
     // Used to store nested collection refs
     mutable ArrayRef m_refs;
+    mutable StringInterner* m_string_interner = nullptr;
```

Comment:
Why do you need to cache the `StringInterner`?  Shouldn't `m_strings` have the interner set?

Reply:
You are right. That should work.